app/assets/stylesheets/pageflow/editor/files_explorer.css.scss

```diff
@@ -64,3 +63,61 @@
     @include icon-button();
   }
 }
+
+.spinner > div {
+  background-color: #ddd;
+  height: 100%;
+  width: 6px;
+  display: inline-block;
+
+  -webkit-animation: loading-animation 1.2s infinite ease-in-out;
+  animation: loading-animation 1.2s infinite ease-in-out;
+}
+
+.loading .spinner {
+  margin: 20px auto;
+  width: 50px;
+  height: 40px;
+  text-align: center;
+  font-size: 10px;
+
+  .rect2 {
+    -webkit-animation-delay: -1.1s;
+    animation-delay: -1.1s;
+  }
+
+  .rect3 {
+    -webkit-animation-delay: -1.0s;
+    animation-delay: -1.0s;
+  }
+
+  .rect4 {
+    -webkit-animation-delay: -0.9s;
+    animation-delay: -0.9s;
+  }
+
+  .rect5 {
+    -webkit-animation-delay: -0.8s;
+    animation-delay: -0.8s;
+  }
+}
```

Comment:
If we have a separate `LoadingSpinnerView`, these rules can be moved into a separate `loading_spinner.css.scss`.